plugins/syntax/plugin_base.dune

```diff
@@ -5,31 +5,23 @@
  (modules g_numeral numeral)
  (libraries coq.plugins.ltac))
 
+(library
+ (name string_notation_plugin)
+ (public_name coq.plugins.string_notation)
+ (synopsis "Coq string notation plugin")
+ (modules g_string string_notation)
+ (libraries coq.plugins.ltac))
```

Comment:
Why do you depend on LTAC here? This seems wrong, it seems you should be able to do with `coq.vernac` as the rest of plugins.